resources/tests/seed.ts

```diff
@@ -30,6 +30,9 @@ export class Seed {
       return {
         invoice_id: faker.string.uuid(),
         created_at: faker.date.past().toISOString(),
+        metadata: {
+          period_end: faker.date.future().toISOString(),
+        },
```

Comment:
I think we are mimicking supabase structure, we can make a flat own structure and in the service map it correctly